scripts/o3de/tests/test_engine_template.py

```diff
@@ -78,6 +78,25 @@ class ${SanitizedCppName}BusTraits
 TEST_CONCRETE_TESTGEM_TEMPLATE_CONTENT_WITH_LICENSE = string.Template(
     TEST_TEMPLATED_CONTENT_WITH_LICENSE).safe_substitute({'SanitizedCppName': "TestGem"})
 
+TEST_TEMPLATE_REPO_JSON = """\
+{
+    "repo_name":"${Name}",
+    "repo_uri": "${RepoURI}",
+    "$schemaVersion": "1.0.0",
+    "origin":"${Origin}",
+    "origin_url":"${OriginURL}",
+    "summary": "${Summary}",
+    "additional_info": "${AdditionalInfo}",
+    "last_updated": "",
+    "gems_data": [],
+    "projects_data": [],
+    "templates_data":[]
+}
+"""
+
+TEST_REPO_JSON = string.Template(TEST_TEMPLATE_REPO_JSON).safe_substitute({'Name': 'TestRepo', 'RepoURI' : "http://test", 'Summary': 'summary',
+                                                                           'Origin': 'o3de', 'OriginURL': "https://github.com/o3de/o3de"})
```

Comment:
nit: might be nicer to keep the line shorter
```suggestion
TEST_REPO_JSON = string.Template(TEST_TEMPLATE_REPO_JSON).safe_substitute({'Name': 'TestRepo',
                                                                           'RepoURI' : "http://test.com",
                                                                           'Summary': 'summary',
                                                                           'Origin': 'o3de',
                                                                           'OriginURL': "https://github.com/o3de/o3de"})
```